### ISSUE TITLE
Does DOT work with an id you indicate? 

### ISSUE THREAD

**Issue by mishav78** [sensity-ai/dot#165]
Can you indicate a picture of an identification card and have it run with a video you indicate? :

 For example you have a picture of a person's driver's license and you supply a pre recorded video, will the program put the face of the driver's license unto the pre recorded video face and then have the person in the video show the driver's license with his or her hand in the video. The same driver's license in the image that was supplied to the program?

**Comment by Ghassen-Chaabouni** [comment]
Yes, it will work if the face in the id is detectable and not too small. If it didn't work, you can increase the size of the id, to make the face more visible
